openeogeotrellis/config/config.py

```diff
@@ -80,7 +80,7 @@ class GpsBackendConfig(OpenEoBackendConfig):
 
     prometheus_api: Optional[str] = os.environ.get("OPENEO_PROMETHEUS_API")
 
-    max_executor_memory: str = "64G"
+    max_executor_or_driver_memory: str = "64G"  # Executors and drivers have the same amount of memory
```

Comment:
@jdries Is this rename ok?
As I understood, driver and executor often have the same specs

Reply:
sure!